src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -33,6 +33,7 @@ class Connection
         'consumer' => 'consumer',
         'auto_setup' => true,
         'delete_after_ack' => false,
+        'delete_after_reject' => false,
```

Comment:
this changes the current behavior, right? 
Should it be `true` to keep the same behavior as before unless explicitly set?

Reply:
Originally it was intended to prevent message loss due, since the default behavior of `ack` is to **not** delete the message.  
It depends how you look at it. From my perspective it's a "bug" because it was an unintended behavior and for this reason should not remove messages by default.

Reply:
Given we are addressing a design issue by introducing a new option, we consider it a new feature hence it targets 5.2. If the feature changes the default behavior, then we consider it a BC break, which is not good :).
Let's keep `true` as default value, if someone comes later on asking for a different default value then we can still consider changing it with some deprecation notices and a smooth upgrade path.

Reply:
btw, we will need a CHANGELOG entry for this feature

Reply:
@chalasr Added a changelog entry and set default value for `delete_after_reject` to `true` as is current behavior.